frontend/src/features/mapSideBar/SidebarContents/LandForm.test.tsx

```diff
@@ -90,10 +90,11 @@ const defaultInitialValues: IParcel = {
   buildings: [],
   evaluations: [],
   fiscals: [],
-  financials: [],
   rowVersion: '',
   latitude: 51,
   longitude: -122,
+  assessedLand: '',
```

Comment:
I'm curious about adding these two?  As they should be covered by `evaluations`.

Reply:
it was just complaining that they weren't there. According to VSCode, you added them to the IParcel three days lol

Reply:
Makes me think we're using the wrong interface in some places.  Those values are flattened values that I'm not certain they would be included in the JSON returned by the endpoint used by this component.  @devinleighsmith may have some insight.

Reply:
These can likely come out, we have some naming issues on some interfaces that I would like to address at some point, essentially there is one IProperty with a flat structure that is used on the property list view, and an IParcel, IBuilding that inherit from a different IProperty that is based on a nested object structure more closely related to how those objects are structured in the DB.

In this case we are using the second type of IProperty, which shouldn't need assessedLand and assessedBuilding. because it gets those values directly from the evaluations array.